vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MergedHeaderItemFactory.kt

```diff
@@ -82,7 +83,7 @@ class MergedHeaderItemFactory @Inject constructor(private val activeSessionHolde
                                                    event: TimelineEvent,
                                                    eventIdToHighlight: String?,
                                                    requestModelBuild: () -> Unit,
-                                                   callback: TimelineEventController.Callback?): MergedMembershipEventsItem_? {
+                                                   callback: TimelineEventController.Callback?): DefaultMergedEventsItem_? {
```

Comment:
OK for the renaming, but maybe something like `SimilarEventsMergedItem` would be more appropriate. `Default` prefix is reserved for default implementation of interfaces.

Reply:
Okay, I renamed the class to `MergedSimilarEventsItem` to be consistent with existing `MergedRoomCreationItem`. Is that okay?